pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -212,19 +212,15 @@ func setProductLabels(cr *api.KieApp, env *api.Environment) {
 func getConsoleName(cr *api.KieApp) string {
 	env := string(cr.Spec.Environment)
 	consoleName := constants.RhpamBusinessCentral
-	if strings.Contains(env, constants.RhdmPrefix) && !isGE713(cr) {
-		consoleName = constants.RhdmDecisionCentral
-	} else if strings.Contains(env, constants.RhpamPrefix) && strings.Contains(env, constants.Production) {
+	if strings.Contains(env, constants.RhpamPrefix) && strings.Contains(env, constants.Production) {
```

Comment:
is still valid the RhpamPrefix with Ithe presence of BMBamoeImagePrefix ?

Reply:
yes, the IBMBamoe is just for image names.